src/Traits/TokenAwareTrait.php

```diff
@@ -29,11 +29,15 @@ public function getToken(): Token
         return $this->token;
     }
 
-    public function getRequestHeaders(): array
+    /**
+     * @return array<string, string>
+     */
+    public function getAuthorizationHeaders(): array
     {
         $tokenType = ucfirst(strtolower($this->token->getTokenType()));
```

Comment:
I'm not 100% its needed, but was thinking to assert that the token is not null before using, methods would fail anyway.. what do you think?